docs/release.md

```diff
@@ -118,3 +118,35 @@ repo][amazon].
 
 [amazon]: https://github.com/aws-quickstart/quickstart-heptio
 
+### 7. Certify the Cluster
+
+Once Amazon has merged the PR, you can certify the cluster. Certification gets
+us a badge that says we're [Certified Kubernetes][certify].
+
+* The main instructions here are in the [`k8s-conformance` repo][certify]. This
+  is the short version.
+* Boot a new cluster using the official [AWS Quickstart][quickstart] site.
+* Set up kubectl using the "outputs" command as in step 5
+* Once it's booted, run the Sonobuoy command on it. `curl -L
+  https://raw.githubusercontent.com/cncf/k8s-conformance/master/sonobuoy-conformance.yaml
+  | kubectl apply -f -`.
+* Tail the logs with `KUBECONFIG=./kubeconfig kubectl logs -f -n sonobuoy
+  sonobuoy`
+* Wait for `MSG="no-exit was specified, sonobuoy is now blocking"`
+* Once it's complete, copy the file off with ` KUBECONFIG=./kubeconfig kubectl
+  cp
+  sonobuoy/sonobuoy:/tmp/sonobuoy/201801241531_sonobuoy_7e807780-bacd-4926-a77c-f24874b38e7d.tar.gz
```

Comment:
i think this should reflect what we have in the sonobuoy readme

`kubectl cp heptio-sonobuoy/sonobuoy:/tmp/sonobuoy ./archive --namespace=heptio-sonobuoy`